Unity-Technologies/ScriptableRenderPipeline
# Expose Camera Capture

### Purpose of this PR
CameraCaptureBridge was created for Unity Recorder but since it uses reflection to pull actions from Recorder there was no way for anyone else to access this functionality.
Since all render pipelines support these actions, it makes sense to simply have the list of actions in render-core and for Unity Recorder and others to set the actions there.

---
### Release Notes
A PR with Unity Recorder is also being submitted.  That project is the only one directly affected by this change.

---
### Testing status

**Manual Tests**: What did you do?
- Tested changes with Unity Recorder to make sure they are still compatible
- Created my own runtime capture that will make use of these exposed actions
---
### Overall Product Risks
**Technical Risk**:  Low

**Halo Effect**:  Low

---
### Comments to reviewers
This PR was the result of an open discussion between the Unity Recorder team and members of all render pipelines.
Initially, we were going to create a new package.  It was much simpler to just make a change to CameraCaptureBridge.
Apologies for whitespace changes.  The only change necessary is CameraCaptureBridge.cs
